src/lib/bootstrap.php

```diff
@@ -19,6 +19,19 @@
 $loader = require __DIR__.'/../app/autoload.php';
 ZLoader::register($loader);
 
+// set current domain for using it as container parameter
+// http://symfony.com/doc/current/cookbook/configuration/external_parameters.html
+if (strpos(__DIR__, '/src/lib') !== false) {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/src/web';
+} else {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/web';
+}
+
```

Comment:
> When only the config line is removed then the code above has no purpose

that parameter is loaded either way, it has default values. By adding the line you are simply changing the defaults. the lines in bootstrap should still work. did you try it?